File: 2015/gara3/pianostudi/att/pianostudi.pas
function pianifica(N: longint; var da, a, crediti: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    pianifica := 42;
end;


const
    MAXN = 100000;
var
    N, i           : longint;
    da, a, crediti : array[0..MAXN-1] of longint;
    fr, fw         : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        readln(fr, da[i], a[i], crediti[i]);

    writeln(fw, pianifica(N, da, a, crediti));
    close(fr);
    close(fw);
end.

File: ed1/gara3/pianostudi/att/pianostudi.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int pianifica(int N, int da[], int a[], int crediti[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


int da[MAXN], a[MAXN], crediti[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(3 == fscanf(fr, "%d%d%d", &da[i], &a[i], &crediti[i]));

    fprintf(fw, "%d\n", pianifica(N, da, a, crediti));
    fclose(fr);
    fclose(fw);
    return 0;
}
